integration/util.go

```diff
@@ -0,0 +1,29 @@
+package integration
+
+import (
+	"fmt"
+	"time"
+)
+
+func retryOperation(timeout time.Duration, interval time.Duration, fn func() error) error {
```

Comment:
This could use a quick godoc to help make things clearer.
```suggestion
// retryOperation will retry the given 'fn' func at the given 'interval', until/unless 'timeout' is reached.
// Any non-nil errors returned by 'fn' in the interim will be ignored.
func retryOperation(timeout time.Duration, interval time.Duration, fn func() error) error {
```